api/label.go

```diff
@@ -70,8 +73,8 @@ type DatabaseLabelUpsert struct {
 
 // LabelService is the service for labels.
 type LabelService interface {
-	// FindLabelKeyList finds all available keys for labels.
-	FindLabelKeyList(ctx context.Context, find *LabelKeyFind) ([]*LabelKey, error)
+	// FindLabelKeys finds all available keys for labels.
+	FindLabelKeys(ctx context.Context, find *LabelKeyFind) ([]*LabelKey, error)
```

Comment:
We actually use xxxList instead of plural form because the latter is inconsistent.

Reply:
I don't see the link explicitly mention to prefer s over List. Also even canonical Go package follows that pattern, I would against their choice.

I would prefer List because I think it's more readable and explicit. English plural has various forms, it's more challenge to get consistency (e.g. We may set a policy to always append s, however that's not always the right syntax, so someone may choose to use the correct plural form).

Also List is more search friendly, we can just do a List regex search to find method returning slices.

 >> The returned slice type indicates the returning if a list so we don't need to use List
 
This is only obvious if looking at the definition. Not so much if just looking from the caller.